setup.py

```diff
@@ -48,7 +48,8 @@
         'Products.GenericSetup',
         'ZODB3',
         'Zope2 >= 2.13',
-        'collective.indexing >= 2.0a2',
```

Comment:
@gforcada removing collective.indexing here is not a good approach in my opinion. This means that all users of Plone < 5.1 have to add collective.indexing as an explicit dependency.